ethereum/test/wormhole.js

```diff
@@ -10,6 +10,13 @@ const testSigner1PK = "cfb12303a19cde580bb4dd771639b0d26bc68353645571a8cff516ab2
 const testSigner2PK = "892330666a850761e7370376430bb8c2aa1494072d3bfeaed0c4fa3d5a9135fe";
 const testSigner3PK = "87b45997ea577b93073568f06fc4838cffc1d01f90fc4d57f936957f3c4d99fb";
 
+const core = "0x00000000000000000000000000000000000000000000000000000000436f7265"
```

Comment:
Might be more readable if
a) we put a comment here saying it's "Core" in ascii
b) construct the string programmatically, like
```
const core = '0x' + Buffer.from("Core").toString("hex").padStart(64,0)
```

Reply:
I like option B, I'll do that.